code/renderers/vue3/template/stories/ReactiveSlots.stories.js

```diff
@@ -0,0 +1,49 @@
+import { expect } from '@storybook/jest';
+import { global as globalThis } from '@storybook/global';
+import { within, userEvent } from '@storybook/testing-library';
+import { UPDATE_STORY_ARGS, STORY_ARGS_UPDATED, RESET_STORY_ARGS } from '@storybook/core-events';
+import { h } from 'vue';
+import BaseLayout from './BaseLayout.vue';
+
+export default {
+  component: BaseLayout,
+};
+
+export const ReactiveSlotTest = {
+  args: {
+    default: 'Default Text Slot',
+    header: () => h('h1', 'Header Functional Component Slot'),
```

Comment:
Could we change this so that the title inner prop is used?

`header: ({title}) => h('h1', title),`

Would be great if we can test that this doesn't regress.